stl/inc/iterator

```diff
@@ -119,24 +131,28 @@ class insert_iterator { // wrap inserts into container as output iterator
 public:
     using iterator_category = output_iterator_tag;
     using value_type        = void;
-    using difference_type   = void;
     using pointer           = void;
     using reference         = void;
+    using container_type    = _Container;

```

Comment:
Why are you grouping `container_type` with the other typedefs in only this iterator? The change is fine, but should be done consistently.

Reply:
I missed reverting this one. I had many more non-functional cleanups here before realizing that they would conflict with #491 and backing them out.